deps/tools/common.mk

```diff
@@ -29,6 +31,7 @@ CMAKE_COMMON += -DCMAKE_CXX_COMPILER="$(CXX_BASE)"
 ifneq ($(strip $(CMAKE_CXX_ARG)),)
 CMAKE_COMMON += -DCMAKE_CXX_COMPILER_ARG1="$(CMAKE_CXX_ARG)"
 endif
+CMAKE_COMMON += -DCMAKE_LINKER="$$(which $(LD))"
```

Comment:
Why are you using `which $(LD)` instead of just `$(LD)`?  Also, it's better to just say `$(shell which $(LD))` instead of using the `$$(...)` syntax.